boards/arm/sama5/sama5d4-ek/scripts/Make.defs

```diff
@@ -44,7 +44,6 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
   ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
 endif
 
-ARCHCPUFLAGS = -mcpu=cortex-a5 -mfpu=vfpv4-d16
```

Comment:
add cortex-a5 to Toolchain.defs

Reply:
It's already handled by Toolchain.defs (Only A8/A9 use vfpv3-d16)